src/web/modules/stripe/model/person.model.js

```diff
@@ -0,0 +1,36 @@
+function buildForTestAccount() {
+  return {
+    first_name: 'Jane',
+    last_name: 'Doe',
+    dob: {
+      day: '01',
+      month: '01',
+      year: '1901'
+    },
+    relationship: {
+      representative: true,
+      executive: true,
+      title: 'CEO'
+    },
+    address: {
+      line1: 'address_full_match',
+      line2: 'WCB',
+      city: 'London',
+      postal_code: 'E1 8QS',
+      country: 'GB'
+    },
+    phone: '8888675309',
+    email: 'test@example.org',
+    verification: {
+      document: {
+        front: 'file_identity_document_success'
+      }
+    }
+  }
+}
+
+export class StripePerson {
```

Comment:
are there any reasons for these to be classes that need to be instantiated? it looks like these could just be 
```
export const stripeTestResponsiblePersonDetails = {
  //...
}
```

This isn't a big problem, it just might not need to be a class

Reply:
Just thought it would be easy to follow classes and objects (and to extend) than making them util files. Happy to change them separately if it looks overkill

Reply:
On second thought, as we don't need classes now just exporting functions makes sense. Updated both account and person models